Content.Server/GameObjects/Components/Mobs/MindComponent.cs

```diff
@@ -137,8 +137,8 @@ public void Examine(FormattedMessage message, bool inDetailsRange)
             if (!HasMind)
             {
                 var aliveText =
-                    $"[color=red]{Loc.GetString("{0:They} {0:are} totally catatonic. The stresses of life in deep-space must have been too much for {0:them}. Any recovery is unlikely.", Owner)}[/color]";
-                var deadText = $"[color=purple]{Loc.GetString("{0:Their} soul has departed.", Owner)}[/color]";
+                    $"[color=purple]{Loc.GetString("comp-mind-examined-catatonic", ("ent", Owner))}[/color]";
+                var deadText = $"[color=red]{Loc.GetString("comp-mind-examined-dead", ("ent", Owner))}[/color]";
```

Comment:
For this message it should actually check MobStatus for dead and whether mind is visiting a ghost. It's out of scope for this PR though.